### PR TITLE
Implement the admin UI modules

### PR DESCRIPTION
**Pull request by bp323**
Implement the admin UI modules per design. Functionality like 'New Module' or 'New series' etc. shouldn't be hooked up yet, we can tackle that in a separate PR.

screen shot 2015-01-16 at 14 37 21

### CODE REVIEW

**Comment by bp323** [comment]
Assigning to @Coenego for review.

File: shared/gh/js/bootstrap.admin-listview.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    $('body').on('click', '.gh-series-select', function() {
+        // Remove the active state from other series
+        $('.gh-series-select').removeClass('gh-series-active');
+        // Add the active state to the clicked series
+        $(this).addClass('gh-series-active');
+
+        // TODO: Load the series in batch edit mode
+    });
+
```
Comment (Coenego) [inline shared/gh/js/bootstrap.admin-listview.js]: Because I can't find anything else to comment on; this empty line can probably go :")

Comment (Coenego) [inline shared/gh/js/bootstrap.admin-listview.js]: I'll fix it in my PR. This is good to go!

Comment (bp323) [inline shared/gh/js/bootstrap.admin-listview.js]: Awesome, thanks for 'fixing' it

**Pull request merged**